src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -101,122 +85,27 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       } recover { case e: Throwable => RequestCompleteWithErrorReport(InternalServerError,
         "Unexpected error updating NIH link", e) }
       // Complete syncWhitelist and ignore as neither success nor failure are useful to the client
-      syncWhiteListResult onComplete {
-        case _ => profileResponse pipeTo parent
-      }
+      syncWhiteListResult pipeTo sender
```

Comment:
This also works:
```
syncWhiteListResult map(Success(_)) recover { case t => Failure(t) } flatMap { _ => profileResponse } pipeTo sender
```
but I'm inclined to think (based on the comment) that this line should just be deleted.